### PR TITLE
Added books.

### PR DESCRIPTION
**Pull request by GintasS**
## What does this PR do?
Add Resources

### Description 
Added two books and a book that is both a book and notes.
Didn't add the 4th book because it was from a unknown source.

### Why is this valuable (or not)?
Because it's free! And about computer graphics too! Unix book is added also!

### How do we know it's really free?`
It is freely hosted on university sites, author sites.

### For book lists, is it a book? For course lists, is it a course? etc.
They're books and a book that is both a book and notes.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by GintasS** [comment]
#3548

**Comment by GintasS** [comment]
**free-programming-books-ta.md**: Removed some things because of Travis CI warnings.

**Comment by eshellman** [comment]
Please add author names where appropriate.

**Comment by GintasS** [comment]
> Please add author names where appropriate.

Added an author for **free-programming-books-ta.md**.

File: free-programming-books.md [hunk 1]
```diff
@@ -284,8 +284,10 @@
 * [Graphics Programming Black Book](http://www.gamedev.net/page/resources/_/technical/graphics-programming-and-theory/graphics-programming-black-book-r1698) - Michael Abrash
 * [Introduction to Modern OpenGL](https://open.gl)
 * [Introduction to TouchDesigner 099](https://leanpub.com/introductiontotouchdesigner/) *(Leanpub account or valid email requested)*
+* [Learn Computer Graphics From Scratch!](https://www.scratchapixel.com) - Scratchapixel
```
Comment (eshellman) [inline free-programming-books.md:287]: at the very least, add an "in progress" note. is this still being worked on?

Comment (GintasS) [reply free-programming-books.md]: Fixed.

**Pull request merged**